src/main/java/com/yahoo/sketches/theta/ConcurrentDirectThetaSketch.java

```diff
@@ -123,9 +129,9 @@
    * Ensures mutual exclusion. No other thread can update the shared sketch while propagation is
    * in progress
    */
-  @Override public void startPropagation() {
-    while (!sharedPropagationInProgress_.compareAndSet(false, true)) {
-    } //busy wait till free
+  @Override
+  public void startPropagation() {
+    while (!sharedPropagationInProgress_.compareAndSet(false, true)) { } //busy wait till free
```

Comment:
Empty blocks are OK.  This formatting is simpler and easier to see that the block is, in fact, empty.